docs/Configuration.md

```diff
@@ -1,130 +1,162 @@
+<!-- DO NOT MODIFY THIS FILE DIRECTLY, IT IS AUTO-GENERATED BY [org.apache.gluten.config.AllGlutenConfiguration] -->
+
 ---
+
 layout: page
 title: Configuration
 nav_order: 3
----
-
-# Spark Configurations for Gluten Plugin
-
-There are many configurations could impact the Gluten Plugin performance and can be fine-tuned in Spark.
-You can add these configurations into spark-defaults.conf to enable or disable the setting.
-
-## Spark parameters
-
-| Parameters                                                    | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Recommend Setting                                    |
-|---------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------|
-| spark.driver.extraClassPath                                   | Gluten Plugin jar file to prepend to the classpath of the driver.                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | /path/to/jar_file                                    |
```

Comment:
It looks like we missed some essential parameters like spark.extraClassPath/spark.plugins/org.apache.spark.shuffle.sort.ColumnarShuffleManager here

Reply:
thanks, these parameters are spark config, i want to create new section to describe the base spark config for gluten.